views/suggestions.html

```diff
@@ -43,46 +42,49 @@ <h1>Share It</h1>
 					title="See the timeline of announcements by Repl.it"
 					>TimeLine</a
 				>
-				<!-- <a
+				<a
 					href="/suggestions"
 					class="suggestions active"
 					title="Leave a suggestion here"
 					>Suggestions</a
-				> -->
+				>
 				<div class="navbar2">
 					<a
-						href="https://github.com/Repl-it-Coders/Repl.it-2020"
+						href="https://github.com/Repl-it-Coders/Repl.it-2021"
 						class="repo"
 						title="View the GitHub repo here!"
 						target="_blank"
 						rel="noreferrer noopener"
 						>GitHub Repo</a
 					>
 					<a
-						href="https://repl.it/@Whippingdot/Replit-2020"
+						href="https://repl.it/@Whippingdot/Replit-2021"
 						class="editor"
 						title="View the project on repl.it"
 						target="_blank"
 						rel="noreferrer noopener"
 						>Project Editor</a
 					>
 					<a
-						href="https://repl.it/@Whippingdot/Replit-2020#README.md"
+						href="https://repl.it/@Whippingdot/Replit-2021#README.md"
 						class="README"
 						title="This it the README.md"
 						target="_blank"
 						rel="noreferrer noopener"
 						>README</a
 					>
+          
+					<a href="{{LOGIN_URL}}" class="login" title="">{{USERNAME}}</a>
```

Comment:
haha pugjs go bbrrrr

Reply:
??

Reply:
haha {x} go brrr meme @whippingdot

Reply:
![image](https://user-images.githubusercontent.com/73612690/104091567-d6563000-5286-11eb-92a5-2662de85a02d.png)

Reply:
But why is he saying that here. And also what does something going brrr mean?